boards/renesas/da1469x_dk_pro/da1469x_dk_pro-pinctrl.dtsi

```diff
@@ -26,20 +26,20 @@
 
 	i2c2_default: i2c2_default {
 		group1 {
-			pinmux = <SMARTBOND_PINMUX(I2C2_SDA, 0, 19)>,
-				<SMARTBOND_PINMUX(I2C2_SCL, 0, 18)>;
+			pinmux = <SMARTBOND_PINMUX(I2C2_SDA, 0, 28)>,
```

Comment:
Can you split these pin control fixes into separate commits? It should be clear in the git history why these are needed